apps/aestratum/src/aestratum_fn.erl

```diff
@@ -0,0 +1,68 @@
+-module(aestratum_fn).
```

Comment:
This is non-standard Erlang style, and different from the rest of aeternity repos... Not ideal and it doesn't improve readability to me.

Reply:
It's aim is to not improve readability, but improve readability of errors.

Normally you get `badmatch` and no surrounding context. 

With `ok_val_err`, you can provide your own error tuple with data.